### ISSUE TITLE
Update guides to use v0.7.0

### ISSUE THREAD

**Issue by dbmikus** [onsip/sipjs.com#15]
The guides should be updated to the latest version. There are some breaking changes between 0.6.x and 0.7.0 and the guides could mislead people.

For example, the "Receive a Call" guide. We must change

``` javascript
userAgent.on('invite', function (session) {
    session.accept({
        media: {
            render: {
                remote: {
                    video: document.getElementById('remoteVideo')
                },
                local: {
                    video: document.getElementById('localVideo')
                }
            }
        }
    });
});
```

to

``` javascript
userAgent.on('invite', function (session) {
    session.accept({
        media: {
            render: {
                remote: document.getElementById('remoteVideo'),
                local: document.getElementById('localVideo')
            }
        }
    });
});
```

There are probably other guides that need to be updated as well.

**Comment by james-criscuolo** [comment]
Should've closed a while ago, guides are currently on 0.13.0

**Comment by dbmikus** [comment]
Thanks, @james-criscuolo.
